mypy/messages.py

```diff
@@ -2297,6 +2297,8 @@ def format_literal_value(typ: LiteralType) -> str:
         if typ.is_noreturn:
             return "NoReturn"
         else:
+            if mypy.options._based:
+                return "Never"
```

Comment:
```suggestion
        else:
            if mypy.options._based:
                return "Never"
```
why even are there 2 different names for the same thing, imo just get rid of one because it doesnt make any sense for the user

Reply:
There still exists `NoReturn` and `Never` idk.

Reply:
I think it's 'okay' for now, maybe not ideal, but better than `<nothing>` imo

Reply:
#372